packages/daimo-common/src/paymaster.ts

```diff
@@ -9,5 +9,5 @@ export type ChainGasConstants = {
   paymasterAddress: Address;
 };
 
-export const DEFAULT_USEROP_VERIFICATION_GAS_LIMIT = 500000n;
+export const DEFAULT_USEROP_VERIFICATION_GAS_LIMIT = 700000n;
```

Comment:
are we taking >500k to verify now?

Reply:
nope, still < 500k but pimlico send needs some margin for unknown reasons (previously too, we were taking <400k but putting 400k would complain)